docs/source/customizations/coprocessor.mdx

```diff
@@ -92,6 +92,20 @@ coprocessor:
 
 In this case, the `RouterService` only sends a coprocessor request whenever it receives a client request. The coprocessor request body includes _no_ data related to the client request (only "control" data, which is [covered below](#coprocessor-request-format)).
 
+### Client configuration
+
+You can control the coprocessor client configuration as needed.  At the moment only experimental_http2 is supported.
```

Comment:
```suggestion
<HttpConnection type="coprocessor" />
```

This should work fine once https://github.com/apollographql/docs/pull/889 merges.

(Tested locally.)